mail_signature_multilang/README.rst

```diff
@@ -0,0 +1,46 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License: AGPL-3
+
+Mail Signature Multilang
+========================
+
+This module was written to add the property translatable to signature field of
```

Comment:
s/to add the property translatable to signature field of users/to make the user signature field translatable.